### PR TITLE
Insertvoltagesource

### PR DESCRIPTION
**Pull request by cbrightsci**

### CODE REVIEW

**Comment by dcwhite** [comment]
@cbrightsci Any test network available? Even if data is private...

**Comment by dcwhite** [comment]
@cbrightsci Still waiting on a test network

**Comment by dcwhite** [comment]
Also needs a new master merge

**Comment by cbrightsci** [comment]
@dcwhite done and done

**Comment by dcwhite** [comment]
Cool, I'll look right now.

File: src/Modules/Legacy/Forward/InsertVoltageSource.cc [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+  if (needToExecute())
+  {
+    auto state = get_state();
+    auto groundFirst = state->getValue(Parameters::GroundFirst).toBool();
+    auto outside = state->getValue(Parameters::InterpolateOutside).toBool();
+    FieldHandle outputField(inputField->clone());
+    DenseMatrixHandle dirichletMatrix;
+    InsertVoltageSourceAlgo algo(groundFirst, outside);
+    algo.ExecuteAlgorithm(voltageSource, outputField, dirichletMatrix);
```
Comment (dcwhite) [inline src/Modules/Legacy/Forward/InsertVoltageSource.cc]: @cbrightsci This is fine for the initial conversion. Can you make an issue called "Refactor InsertVoltageSource to use generic algorithm mechanism" and assign to yourself, milestone 2016.

**Pull request merged**